Dockerfile

```diff
@@ -23,21 +23,7 @@ WORKDIR /app
 COPY . .
 
 # Precompile tmail flutter
-RUN cd core \
-  && flutter pub get \
-  && cd ../model \
-  && flutter pub get && flutter pub run build_runner build --delete-conflicting-outputs \
-  && cd ../contact \
-  && flutter pub get && flutter pub run build_runner build --delete-conflicting-outputs \
-  && cd ../rule_filter \
-  && flutter pub get && flutter pub run build_runner build --delete-conflicting-outputs \
-  && cd ../forward \
-  && flutter pub get && flutter pub run build_runner build --delete-conflicting-outputs \
-  && cd .. \
-  && flutter pub get && flutter pub run build_runner build --delete-conflicting-outputs \
-  && flutter pub get && flutter pub run intl_generator:extract_to_arb --output-dir=./lib/l10n lib/main/localizations/app_localizations.dart \
-  && flutter pub get && flutter pub run intl_generator:generate_from_arb --output-dir=lib/l10n --no-use-deferred-loading lib/main/localizations/app_localizations.dart lib/l10n/intl*.arb \
-  && flutter build web --profile
+RUN bash prebuild.sh && flutter build web --profile
```

Comment:
Can't see prebuild.sh. Is that normal?

Reply:
I'm not sure what you meant? The prebuild.sh file is in the repo [here](https://github.com/linagora/tmail-flutter/blob/master/prebuild.sh)